### PR TITLE
Add vote creation

### PR DESCRIPTION
**Pull request by mariogarcia**
- [ ] Create a vote with a group allowing anonymous vote
- [ ] Create a vote with a group `NOT` allowing anonymous vote

### CODE REVIEW

File: src/components/shared/VoteForm/VoteForm.css [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+  }
+
+  & .score {
+    & label {
+      align-items: center;
+      border-radius: 10%;
+      color: var(--white-500);
+      cursor: pointer;
+      display: flex;
+      font-size: 2rem;
```
[the remark this answers is not in the document]
Comment (mariogarcia) [reply src/components/shared/VoteForm/VoteForm.css]: Applying `font-size: calc(5rem * var(--vote-font-size-factor));` make the font so tiny:

with_font_calculated

With current `2rem` font-size looks like the actual design at [figma](https://www.figma.com/file/NRpya9xCxkxHZ7WjvIkDJskt/Kalhappy?node-id=26%3A49)

with_rem_font

So I don't know, what do you think ? Do we use `calc` or do we stick to `2rem` here ?

File: src/components/shared/VoteForm/VoteForm.css [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+      }
+
+      &:not(:checked) + label {
+        opacity: .2;
+      }
+    }
+  }
+
+  & .submit {
+    font-size: calc(20 * var(--vote-font-size-factor));
```
[the remark this answers is not in the document]
Comment (mariogarcia) [reply src/components/shared/VoteForm/VoteForm.css]: Done!

File: src/components/shared/VoteForm/VoteForm.css [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+
+    & input[type="radio"] {
+      opacity: 0;
+
+      &:checked + label,
+      &:focus + label {
+        opacity: 1;
+        transform: scale(1.2);
+      }
+
```
[the remark this answers is not in the document]
Comment (mariogarcia) [reply src/components/shared/VoteForm/VoteForm.css]: Done

File: src/locales/es.json [hunk 4]
```diff
@@ -97,6 +101,9 @@
         "PARAGRAPH": "¿Todavía no tienes una cuenta?",
         "BUTTON": "Regístrate"
       }
-    }
+    },
+    "VOTE": {
+     "TITLE": "Como te encuentras hoy ?"
```
[the remark this answers is not in the document]
Comment (mariogarcia) [reply src/locales/es.json]: Done

File: src/components/shared/VoteForm/VoteForm.css (hunk 1 above)
[the remark this answers is not in the document]
Comment (bameda) [reply src/components/shared/VoteForm/VoteForm.css]: Ok

**Pull request merged**